Task 1/helpers/helper.js

```diff
@@ -0,0 +1,3 @@
+export function transformAndDeleteCharacters(string) {
```

Comment:
Bad naming. You have a dollarsString on parameter, and number on return. `convertDollarsToNumber` - is better